react/QuantitySelector.tsx

```diff
@@ -31,6 +31,8 @@ const QuantitySelector: FunctionComponent = () => {
         maxValue={MAX_ITEM_QUANTITY}
         onChange={onQuantityChange}
         disabled={item.availability !== AVAILABLE}
+        unitMultiplier={item.unitMultiplier ?? undefined}
+        measurementUnit={item.measurementUnit ?? undefined}
```

Comment:
```suggestion
        unitMultiplier={item?.unitMultiplier}
        measurementUnit={item?.measurementUnit}
```

Reply:
this won't work, because the expected type for these props are `string | undefined`, but the type of `item?.unitMultiplier` is `string | undefined | null`

Reply:
oh, unitMultiplier can be null?

Reply:
it can the way the types are on our graphql :-/